### DISCUSSION TITLE
Restrict access to MySQL | max_connections_per_hour

### DISCUSSION THREAD

**Discussion by Mabdulkhalek** [kanboard/kanboard#5211] [Installation & Configuration]
Dears,

Appreciate your support and advise as i have received this email from my hosting company and they temporarily restrict access to MySQL for my database, i can't access my kanboard now receiving the following error

Internal Error: SQLSTATE[HY000] [1226] User 'xxxx123' has exceeded the 'max_connections_per_hour' resource (current value: 1)

**Received Email from Hostgator:**

Hello,

Unfortunately, we have been forced to temporarily restrict access to MySQL for dddd333 on xxxx.hostgator.com. Please take a moment to review this email in full as it contains important information and resources to assist you in resolving this issue. Please note that while this restriction is temporary, you should take actions to resolve the issue(s) which may have caused it in order to avoid further down-time/permanent restrictions.

- How can you resolve the issue?
We have two solutions available. Moving the account to a dedicated server, which will allow far greater hardware resources or you may try following the basic recommendations for optimizing your database. If you feel your database has already been optimized, it may simply be time to consider the hardware upgrade. http://support.hostgator.com/articles/specialized-help/technical/optimize-mysql-resource-usage

- Why did this happen?
Per our terms of service (http://www.hostgator.com/tos) a single hosting account may use no more than 25% of the entire server's resources. Accounts are typically not actively restricted until they exceed those resource limitations exceptionally. Unfortunately, in this instance, we were forced to place a temporary restriction on your user to prevent service issues with the server's over-all system performance. For more information about resource usage limitations, please see:https://support.hostgator.com/articles/pre-sales-policies/rules-terms-of-service/account-limits-summary

Please reply to this email once you have determined a resolution to this issue.

CPU_TIME:14 table_rows_read:11262 SELECTS:1072 ROWS_UPDATED:125 ROWS_FETCHED:4244 BUSY_TIME:14 ONNECTED_TIME:31 BYTES_SENT:643586132 BYTES_RECEIVED:661445944 WAIT_TIME:0

Top table row reads:
DB_USER: xx123-- TOTAL_CONNECTIONS: 87 -- CONNECTED_TIME: 31 -- CPU_TIME: 14 -- TABLE_ROW_READS: 11262 -- SELECT_COMMANDS: 1072 -- UPDATE_COMMANDS: -- BUSY_TIME: 14 -- BYTES_SENT: 643586132 -- BYTES_RECEIVED: 661445944 -- WAIT_TIME (IO): 0

Top WAIT (IO) TIME:
DB_USER: xxx123 -- TOTAL_CONNECTIONS: 87 -- CONNECTED_TIME: 31 -- CPU_TIME: 14 -- TABLE_ROW_READS: 11262 -- SELECT_COMMANDS: 1072 -- UPDATE_COMMANDS: -- BUSY_TIME: 14 -- BYTES_SENT: 643586132 -- BYTES_RECEIVED: 661445944 -- WAIT_TIME (IO): 0

Thu Apr 7 04:15:18 CDT 2022
Running Processes:

Running Queries:

Open connections

Current Site Requests:

**Comment by Lewnidas** [comment]
Did you solve this issue? I have the exact same error with Hostgator.

**Comment by Mabdulkhalek** [comment]
Not yet, i didn't apply any redemption action.. just solved by itself after an hour..Need to know the root cause to avoid its repetitionOn 13/04/2022, 16:45 Lewnidas ***@***.***> wrote:

   
  Did you solve this issue? I have the exact same error with Hostgator.

**Comment by fguillot** [comment]
This PR #5042 might help. Otherwise, ask your hosting provider to increase the number of connections.
